demo/src/public/experiment-page2.js

```diff
@@ -1,14 +1,34 @@
 (function() {
     'use strict';
 
-    var lou = new Mozilla.TrafficCop({
+    var variants = {
+        'a': 30,
+        'b': 30,
+        'c': 30
+    };
+
+    function handleVariation(variation) {
+        // wait until DOM is ready to be manipulated...
+        domReady(function() {
+            // make sure variation is one we are expecting (and not noVariationCookieValue)
+            if (variants.hasOwnProperty(variation)) {
+                var target = document.getElementById('var-' + variation);
+                target.classList.remove('hidden');
+            }
+        });
+    }
+
+    // non-jquery document ready function
+    // http://beeker.io/jquery-document-ready-equivalent-vanilla-javascript
+    function domReady(callback) {
```

Comment:
My major reason for keeping this out of TC is because it's only IE 9 and above. If you need to target IE 8 and lower, the function is quite a bit longer. I didn't feel comfortable forcing either decision into the lib.

On a lesser note, I didn't want to force the implementor to wait for DOM ready if they don't need it. Maybe another JS/CSS file will be loaded for a variation? This is admittedly speculative.

Overall, I'm trying to keep TC focused on variations and not what happens after one is chosen. Happy to hear another perspective though.